uber/utils.py

```diff
@@ -198,7 +202,7 @@ def to_dict(self):
 
     @property
     def models(self):
-        return [self.from_sessionized(d) for d in self.targets]
+        return self._models_cached
```

Comment:
I realized that like, right after. Since we just tested this a bunch and I'd like to limit churn, I'll just make an issue for now